activemq-client/src/main/java/zipkin2/reporter/activemq/ActiveMQSender.java

```diff
@@ -102,7 +102,7 @@ public Builder encoding(Encoding encoding) {
       return this;
     }
 
-    /** Maximum size of a message. Default 1000000. */
+    /** Maximum size of a message. Default 500KB. */
     public Builder messageMaxBytes(int messageMaxBytes) {
```

Comment:
Do you think we should update these builder methods to validate the size is less than the implementation limit (e.g., 1MB for kafka, 2GB for activemq)?

Reply:
not sure, as this limit can be customized (in Kafka at least)